test/manual/definitions/apm/php-agent/php-apache-wordpress-linux2.json

```diff
@@ -0,0 +1,29 @@
+{
+    "global_tags": {
+        "owning_team": "OpenSource",
+        "Environment": "development",
+        "Department": "Product",
+        "Product": "Virtuoso"
+    },
+
+    "resources": [{
+      "id": "php-a-w-l2",
+      "display_name": "PHP Apache Wordpress",
+      "provider": "aws",
+      "type": "ec2",
+      "size": "t3a.micro",
+      "ami_name": "amazonlinux-2-base*",
+      "user_name": "ec2-user"
+    }],
+
+    "services": [
+        {
+          "id": "wordpress",
+          "display_name": "RedHat PHP Apache Wordpress",
+          "local_source_path": "/mnt/open-install-library",
```

Comment:
@Fahmy-Mohammed there are a few more files that need to have the local_source_path updated